server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -55,9 +59,20 @@ protected RestChannelConsumer prepareRequest(final RestRequest request, final No
     }
 
     AddVotingConfigExclusionsRequest resolveVotingConfigExclusionsRequest(final RestRequest request) {
-        String nodeName = request.param("node_name");
+        String nodeDescriptions;
+
+        // TODO This request param is being deprecated
```

Comment:
See e.g. here for how to emit deprecation warnings:

https://github.com/elastic/elasticsearch/blob/09b46c86463a3628dd216e33d23c5fb9495a047c/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestSyncedFlushAction.java#L65-L66

Reply:
Added deprecation warning.